.github/actions/use-pre-commit/action.yml

```
@@ -18,11 +18,22 @@ inputs:
     description: pre-commit version to use
     required: false
     default: 2.21.0
+  install-only:
+    description: Only install pre-commit without running it.
+    required: false
+    default: "false"
```

Use single quotes to be consistent with other strings?
```suggestion
    default: 'false'
```

---

Maybe we should enforce that in `yamllint` configuration ?

---

Why not... will it impact a lot of yaml files? 

Btw, someone seem to prefer double quotes :smile: 
https://github.com/Scille/parsec-cloud/blob/c009a160434cb1edae8806caa8388389d6f08147/.yamllint.yml#L77-L78

---

Yes that's me :sweat_smile: 
Yes it will impact some yaml file but that will be in a specific PR to simplify that